libraries/chain/webassembly/privileged.cpp

```diff
@@ -183,7 +183,7 @@ namespace eosio { namespace chain { namespace webassembly {
          EOS_ASSERT( f.description.size() <= config::max_finalizer_description_size, wasm_execution_error,
                      "Finalizer description greater than ${s}", ("s", config::max_finalizer_description_size) );
          f_weight_sum += f.fweight;
-         std::optional<bls12_381::g1> pk = bls12_381::g1::fromJacobianBytesLE(f.public_key_g1_jacobian);
+         std::optional<bls12_381::g1> pk = bls12_381::g1::fromAffineBytesLE(f.public_key_g1_affine_le);
```

Comment:
The default `raw` parameter for `romAffineBytesLE` is false. You need to set it to true as we are using raw format.

Reply:
What does `check` do?